src/sqlfluff/cli/commands.py

```diff
@@ -405,7 +405,7 @@ def lint(
             click.echo(cli_table(timing_summary[step].items()))
 
     if not nofail:
-        if not non_human_output:
+        if not non_human_output and not config.get("nocolor"):
```

Comment:
Shouldn't this have some type of place holder even if `nocolor` is selected?

Thinking something like:
```python
if not non_numan_output:
  if not config.get("nocolor"):
    click.echo("All Finished 📜 🎉!!")
  else:
    click.echo("All finished!")
```

@barrywhart I love that someone got to this feature!

Reply:
Great point! Making that change now...